src/setup_utilities/state-management.ts

```diff
@@ -26,9 +26,26 @@ import { loadProjectsFromFile, setWorkspaceSettings, generateGitIgnore, generate
 export async function loadGlobalState(context: vscode.ExtensionContext): Promise<GlobalConfig> {
   let globalConfig: GlobalConfig = await context.globalState.get("zephyr-ide.state") ?? {
```

Comment:
Will this work if globalConfig doesn't have all the fields. Should the type be any and casted to globalConfig after the deprecation check?